main.cc

```diff
@@ -1967,7 +1967,7 @@ int main(int ac, char** av) {
         return 1;
     }
 
-    // Even on the environment which causes error during initalize Scylla,
+    // Even on the environment which causes error during initialize Scylla,
```

Comment:
nit: the typo is fixed, but the it still isn't proper English. Should be perhaps 'Scylla initialization' and instead of 'the environment' it better be 'an environment'